### PR TITLE
chore: rename file .tsx => .ts

### PR DESCRIPTION
**Pull request by li-jia-nan**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [x] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | 无需纳入ChangeLog |
| 🇨🇳 Chinese | 无需纳入ChangeLog |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/popover/PurePanel.tsx [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
 import useStyle from './style';
 
 export const getOverlay = (
-  prefixCls: string,
+  prefixCls?: string,
   title?: PopoverProps['title'],
   content?: PopoverProps['content'],
 ) => {
-  if (!title && !content) return undefined;
+  if (!title && !content) {
```
Comment (li-jia-nan) [inline components/popover/PurePanel.tsx:15]: 这里换行之后 cov 掉了，顺便在下面补了个 test case 👇

**Review by yoyo837 — APPROVED** [review]

File: components/popover/PurePanel.tsx (hunk 1 above)
Comment (yoyo837) [reply components/popover/PurePanel.tsx:15]: 没看见补的test case呢？

**Comment by li-jia-nan** [comment]
image

**Comment by li-jia-nan** [comment]
@yoyo837 cc

**Comment by yoyo837** [comment]
后面看到了

**Comment by afc163** [comment]
好像没必要改这个。

**Pull request merged**